parking

```diff
@@ -0,0 +1,97 @@
+package javaapplication1;
+import java.util.*; 
+
+class Vehicle{
+	int wheels, type_int;
+	String plate_number, in_time, type_string;
```

Comment:
Use objects instead of primitive types.

change it to this
private Integer numberOfWheels;
private Integer typeInt; 
private String numberPlate;
private String inTime;
private String type;